.test/app-config.ts

```diff
@@ -1,13 +0,0 @@
-/**
- * Copyright (c) HashiCorp, Inc.
- * SPDX-License-Identifier: MPL-2.0
- */
-
-import path from 'path'
-import { unflatten } from 'flat'
-import { getHashiConfig } from '../config'
-
-const env = process.env.HASHI_ENV || 'development'
-const envConfigPath = path.join(process.cwd(), 'config', `${env}.json`)
-
-export const __config = unflatten(getHashiConfig(envConfigPath))
```

Comment:
This was only ever used in one test, [src/__tests__/e2e/seo.spec.ts](https://github.com/hashicorp/dev-portal/pull/2853/files#diff-65543d7399a3526230ded80d17ae2289cca6d669b8d8a74195a56a326159592d), so we moved it into that test.